unstructured/file_utils/filetype.py

```diff
@@ -747,13 +747,13 @@ def _file_type(self) -> FileType | None:
 
             filenames = zip.namelist()
 
-            if "word/document.xml" in filenames:
+            if any(re.match(r"word/document.*\.xml$", filename) for filename in filenames):
```

Comment:
I think the final word on `is_docx` will be a package part with the `ContentType` you highlighted in the `.rels` file, but I'm fine with this as a step toward that :)